tests/unit/streamalert/scheduled_queries/command/processor_test.py

```diff
@@ -0,0 +1,108 @@
+"""
```

Comment:
I noticed all the unit test files have `_test` as suffix except last unit test file `tests/unit/streamalert/scheduled_queries/streamalert/test_kinesis.py`. All existing streamalert unit test files have `test_` as filename prefix.

Reply:
Omg I thought "fixed" this but in reality I actually flipped it around 🤕

Reply:
Fixed